doc/classes/Control.xml

```diff
@@ -620,6 +646,7 @@
 				Forwarding can be implemented in the target control similar to the methods [method get_drag_data], [method can_drop_data], and [method drop_data] but with two differences:
 				1. The function name must be suffixed with [b]_fw[/b]
 				2. The function must take an extra argument that is the control doing the forwarding
+				See [method get_drag_data_fw], [method can_drop_data_fw], [method drop_data_fw].
```

Comment:
The original documentation can be removed/reworked as it described how to manually come to the prototype for the newly bound methods. Instead, it should now tell users that they should implement those three virtual methods.